templates/web/base/report/_get_assigned_user.html

```diff
@@ -0,0 +1,6 @@
+[% IF problem.user_planned_reports.user.name OR problem.shortlisted_user %] [%# this report is on a shortlist %]
```

Comment:
I don't understand what's happening here, sorry. Why is checking `problem.shortlisted_user` not enough?